scoap3/modules/theme/templates/scoap3_theme/_navbar.html

```diff
@@ -22,6 +22,7 @@
     <ul class="nav navbar-nav">
       <li><a href="/">Home</a></li>
       <li><a target="_blank" href="https://scoap3.org/">SCOAP<sup>3</sup> project</a></li>
+      <li><a href="/partners">For partners</a></li>
```

Comment:
Only "Partners" would be more consistent IMO

Reply:
For me a Partners link would mean that there will be some information about the partners. But here we provide information useful **for** the partners and not about them.